app/src/renderer/components/wallet/PageWallet.vue

```diff
@@ -23,6 +23,12 @@
       class="tm-li-balance"
       @show-modal="showModal"
     />
+    <tm-btn
```

Comment:
this button should go on line 10 with `slot="header-buttons"`

Reply:
like how we do it with create proposal

Reply:
good easy solution. I proposed a data message with a button and an explanation of what the faucet is.

Reply:
Thanks! just pushed the slottized button, I haven't thought of this

Reply:
hmm, I still don't think this is a sufficient solution. created a follow up: https://github.com/cosmos/voyager/issues/2134

Reply:
Yeah I was thinking how to make it look cool but I have little ideas on what to write tht is short and descriptive too

Reply:
please still do this: 

line 10 with `slot="header-buttons"`

Reply:
I've put the slot down here, line 28

Reply:
@jbibla he is correct (I checked). It just doesn't show here on GitHub.